src/checkbox/index.ts

```diff
@@ -21,3 +23,5 @@ export { STATE_TYPE, STYLE_TYPE, LABEL_PLACEMENT } from './constants';
 
 // Flow
 export * from './types';
+/** @deprecated use CheckboxReducerState instead. To be removed in future versions.*/
+export type initialState = CheckboxReducerState;
```

Comment:
```suggestion
export type StateT = CheckboxReducerState;
```

Reply:
this is alias to replace old type definition from here:  https://github.com/uber/baseweb/blob/master/legacy-dts/checkbox/index.d.ts#L21-L24 so initialState is indeed a correct name, but I just noticed it is not actually exported… -- will remove this